sys/linux/socket.txt

```diff
@@ -317,6 +321,41 @@ ifr_ifru [
 
 define IFREQ_SIZE	sizeof(struct ifreq)
 
+# https://elixir.bootlin.com/linux/v5.3.11/source/include/uapi/linux/if_vlan.h
+vlan_args {
+	cmd		flags[vlan_cmd, intptr]
```

Comment:
s/intptr/int32/
The structure contains int. Int in C is 32 bits always on relevant ABIs. Intptr is 64-bits on 64-bit arches.

Reply:
Updated to int32 here and all the following ints. Just want to confirm one thing that is the long and ulong the only numeric value we use intptr to present it?

Reply:
Well... they are the main ones... I can't make any definitive statements about all gazillion types used in kernel...
char/short/int must _not_ be intptr
But I am sure there are some other typedefs that are pointer-sized too